### PR TITLE
copy edits, fixed broken links. reworked autocomplete lesson, make it cl...

### PR DESCRIPTION
**Pull request by galiat**
...ear to use the VM for these steps

### CODE REVIEW

File: sites/en/installfest/checklist.md [hunk 1]
```diff
@@ -1,22 +1,27 @@
 # Installfest Completion Checklist
 
-1. Open a command line, then go to the workspace folder : type `cd ~/workspace`
+1. Open a command line on your computer.
+  - **On Macs:** In Finder, start Applications > Utilities > Terminal, or find the Terminal application through Spotlight (click the magnifying glass in the top right of the screen and start typing 'Terminal')
+  - **On Windows:** open up **Git Shell** from your desktop or All Programs menu. Do not use Power Shell! It doesn\'t support the ssh command that we need.
+  - **On Linux:** press Ctrl + Alt + T or Find Terminal under the Accessories category of your applications menu.
 
-2. Start the virtual machine: type `vagrant up`
+1. Go to the workspace folder: type `cd ~/workspace`
```
Comment (eudaimonious) [inline sites/en/installfest/checklist.md:8]: Do the correct numbers get completed for you or are all of these 1's a mistake?

Comment (galiat) [inline sites/en/installfest/checklist.md:8]: correct numbers would be shown.

~Galia

**Comment by eudaimonious** [comment]
LGTM

**Comment by mxie** [comment]
This looks fine to me.

**Pull request merged**